tests/libcxx/expected_results.txt

```diff
@@ -61,18 +53,36 @@ std/language.support/support.limits/support.limits.general/string_view.version.p
 # libc++ doesn't yet implement P2231R1, so it expects an old value for `__cpp_lib_optional`
 std/language.support/support.limits/support.limits.general/optional.version.pass.cpp FAIL
 
-# test emits warning C4310: cast truncates constant value
-std/numerics/bit/bitops.rot/rotl.pass.cpp:0 FAIL
-
 # libc++ doesn't yet implement P1754R1 or P1964R2, so it expects an old value for `__cpp_lib_concepts`
 std/language.support/support.limits/support.limits.general/concepts.version.pass.cpp FAIL
 
-# Bogus test believes that optional<non_constexpr_destructor> cannot be a literal type
-std/utilities/optional/optional.object/optional.object.dtor/dtor.pass.cpp:0 FAIL
-
 # Bogus test believes that copyability of array<T, 0> must be the same as array<T, 1>
 std/containers/sequences/array/array.cons/implicit_copy.pass.cpp FAIL
 
+# string_view and array iterators are not portably pointers (https://reviews.llvm.org/D117368)
+std/iterators/iterator.primitives/iterator.traits/cxx20_iterator_traits.compile.pass.cpp FAIL
+std/iterators/iterator.requirements/iterator.cust/iterator.cust.move/iter_move.pass.cpp FAIL
+std/strings/string.view/string.view.cons/from_iterator_sentinel.pass.cpp FAIL
+std/strings/string.view/string.view.deduct/iterator_sentinel.pass.cpp FAIL
+std/utilities/format/format.formatter/format.parse.ctx/advance_to.pass.cpp FAIL
+std/utilities/format/format.formatter/format.parse.ctx/begin.pass.cpp FAIL
+std/utilities/format/format.formatter/format.parse.ctx/ctor.pass.cpp FAIL
+std/utilities/format/format.formatter/format.parse.ctx/end.pass.cpp FAIL
+
+# libc++ doesn't correctly constrain the iterator_traits specialization for common_iterator (https://reviews.llvm.org/D117449)
+std/iterators/predef.iterators/iterators.common/iterator_traits.compile.pass.cpp FAIL

```

Comment:
This was merged - regen? (I am definitely OK with deferring this until the next update.)